### ISSUE TITLE
Make the Welcome Page Responsive

### ISSUE THREAD

**Issue by ezioda004** [UdacityFrontEndScholarship/quiz-app#65]
## Make the Welcome Page Responsive

The images arent scaling properly and the sections can be made to be stack on one another rather than on side with the images for mobile.

![screenshot 69](https://user-images.githubusercontent.com/20020273/39095386-6ae1dfe6-465d-11e8-845a-f22a2cffafe3.png)
![screenshot 70](https://user-images.githubusercontent.com/20020273/39095389-6dfd0d04-465d-11e8-8735-efec89136325.png)

How to get started? Read our [project setup instructions](https://github.com/UdacityFrontEndScholarship/quiz-app/blob/master/CONTRIBUTING.md#project-setup-instructions).
Fork the project, navigate to directory 
`quiz-app => src => Component => Home => Home.js and Home.css`

You may also need to style the main container component. Its located in 
`quiz-app => src => Component => Welcome Page =>Welcome.css `

We're using Bootstrap-4 for this project but feel free to use native CSS wherever you require.

Note: 
1. If you use any self closing tags such as `<img>`, `<input>`, `<br>`
Make sure you always close them like this `<img />`, `<input />`, `<br />`

2. If you add a `class` to an element example 
```
<div  class = "nav navbar"> 
    <!-- rest of your code -->
</div>
```
Make sure you use **className** instead of **class**. Example

```
<div  className = "nav navbar"> 
    <!-- rest of your code -->
</div>
```

Feel free to ask any question regarding this to us here or over slack.

Comment on this issue and we'll assign this to you.

**Comment by goss-gowtham** [comment]
"stack on one another", Is this phrase mean that Image then texts below the other?
Can I work on this Issue? @ezioda004

**Comment by ezioda004** [comment]
Hi @goss-gowtham,
>stack on one another" 

Yes, the images and the sections text show be below each other.
 
You already have an open PR, I suggest fixing that first. I cannot assign this to you until you fix that.

**Comment by abhishekakade** [comment]
Can I work on this?

**Comment by sb39** [comment]
hey @ezioda004, assign it to me , and give me a brief , so that it becomes a bit easy to work upon, Thanks

**Comment by kadumuri1994** [comment]
@sb39 , you can start working on this issue. Try reading the requirement in the issue. If you still face issue in understanding, you can drop your questions here  .
 @SukhjinderArora  and @goss-gowtham , as you have already worked on few issues  we are giving a chance to first timers. Hope you understand :)

**Comment by ezioda004** [comment]
@sb39 You can work on this. Please contact me on slack if you're unable to understand anything.
As @kadumuri1994 said we're giving first timers priority. We'll be making more issues so more will get chance.

**Comment by sb39** [comment]
@ezioda004 please can you brief the problem and i would wrap it up by tommorow

**Comment by abhishekakade** [comment]
@sb39, read the issue. Everything has been explained there. The images aren't scaling properly and the sections can be made to be stack on one another rather than on side with the images for mobile. The images and the sections text should be below each other.
Means in mobile view, it should look like **image** and then the **text should be below the image** rather than besides it like in the image in issue above.

**Comment by abhi1A** [comment]
I would like to work on this issue as its still not closed yet. Guys who are working on this if I can work on the problem if you are not proceeding forward with it. @ezioda004 can you brief a little more about the problem?

**Comment by ezioda004** [comment]
Resolved #69 
Closing this.
